cfme/common/provider_views.py

```diff
@@ -576,3 +576,14 @@ def before_fill(self, values):
         for widget in self.COND_WIDGETS:
             if values.get(widget):
                 getattr(self, widget).fill(values.get(widget))
+
+
+class TemplatesCompareView(InfraProvidersView):
+    """Compare Templates page."""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+
+    @property
+    def is_displayed(self):
+        title = "Compare VM Template and Image"
+        return (self.logged_in_as_current_user and self.navigation.currently_selected == [
```

Comment:
_optional_ `self.logged_in_as_current_user` not necessary as it's covered by parent views.

Reply:
Removed as requested.